kratos/python_scripts/gid_output_process.py

```diff
@@ -645,3 +653,48 @@ def __remove_list_files(self):
                         os.remove(f)
                     except WindowsError:
                         pass
+                    
+    #
+    def __remove_post_results_files(self, step_label):
```

Comment:
Yes you are right, it must be only a single underline for an internal method. I was following the notation or style found in the script. Maybe some of the internal methods have to be revised too. @jcotela